optionals/compat_rhs_afrf3/CfgVehicles.hpp

```diff
@@ -282,14 +282,28 @@ class CfgVehicles {
     class Helicopter_Base_F;
     class Helicopter_Base_H: Helicopter_Base_F {
         class EventHandlers;
+        class Turrets;
+    };
+    class Heli_Light_02_base_F: Helicopter_Base_H {
+        class Turrets: Turrets {
+                class CopilotTurret;
+                class MainTurret;
```

Comment:
`Heli_Light_02_base_F` does not have `MainTurret` in `class Turrets` in vanilla. This causes potential problems with other helis inheriting this `Turrets` passively by introducing an invisible gunner slot.
If this really is in RHS this way, then it makes no difference. But we should make sure that this is a RHS bug and not us introucing it with the compat PBO.

```
    class Heli_Light_02_base_F: Helicopter_Base_H {
        class Turrets: Turrets {
            class CopilotTurret: CopilotTurret {
            };
        };
    };
```